tests/Unit/Booking/BookingsServiceTest.php

```diff
@@ -34,7 +35,7 @@ protected function setUp(): void
 
         $this->service = $this->app->make(BookingsService::class);
 
-        $this->mockUserModel = User::initModelWithData([
+        $this->mockUserModel = new User([
             'id' => 1234567,
             'name_fist' => 'First',
```

Comment:
Does this misspelling of 'name_fist' affect anything?

Reply:
You're correct, thanks!